README.md

```diff
@@ -94,11 +94,12 @@ In Azure Synapse notebooks please place the following in the first cell of your
 {
   "name": "synapseml",
   "conf": {
-      "spark.jars.packages": "com.microsoft.azure:synapseml_2.12:0.10.2",
```

Comment:
Is this the only place where this change is needed? Theres a few install instruction blocks lurking around the codebase